hexrdgui/masking/constants.py

```diff
@@ -0,0 +1,7 @@
+class MaskType:
+    region = 'region'    # Rectangle or Ellipse mask
+    polygon = 'polygon'  # Hand drawn mask
+    threshold = 'threshold'
+    powder = 'powder'
+    laue = 'laue'
+    pinhole = 'pinhole'
```

Comment:
We need to know the type of mask for the mask boundaries feature. I chose to be explicit about each type in case we have need for these details in future feature requests, but if we want to keep things simpler we could just have something like `threshold`, `computed` (`powder`, `laue`, `pinhole`) and `drawn` (`region`, `polygon`)? I'm open to better names of course (`threshold` could arguably be included in `computed` as well).

Reply:
We can keep these for now! Thanks Brianna.